package.json

```diff
@@ -45,5 +45,10 @@
     "react": "^15.5.1",
     "uglifyify": "^3.0.4",
     "uglifyjs": "^2.4.10"
+  },
+  "browserify": {
+    "transform": [
+      "loose-envify"
```

Comment:
That's a good question. Seems like yes. I see it in React's dependencies as well.